### DISCUSSION TITLE
💡3차 세미나에서 새롭게 알고싶은 혹은 아주 어려웠던 내용을 나눠봅시다!💡

### DISCUSSION THREAD

**Discussion by sohyundoh** [NOW-SOPT-SERVER/seminar#5] [General]
3차 세미나 따라오기 어려웠다면--? 삐빕 정상입니다..‼️
어려웠던 내용을 마구마구 남겨주세요! 에러가 났던 내용도 도움을 얻고싶다면 남겨주셔도 좋습니다!! 다 같이 성장해보자구요!

##  3차 세미나 내용이나 과제에서 새롭게 배운점을 자유롭게 공유해봅시다!
이런 내용은 파트원들과 공유해보고 싶다! 라는 내용을 마음껏 공유해주세요!
파트원들과 나누고 싶은 질문이 있다면 남겨주세요!

## 3차 세미나가 끝나고 새로 배워보고 싶은 내용이 있다면 남겨주세요!
파트장이 세미나를 준비하는데 도움을 얻을 수 있게 새로 배워보고 싶은 내용을 마구마구 남겨주세요!
Discussion을 적극 활용해 새로운 Discussion을 직접 생성하고 질문하는 경험을 가져보는게 어떨까요?

**Comment by geniusYoo** [comment]
쓰기 지연 SQL 저장소와 1차 캐시의 개념이 약간 헷갈립니다 ,,

영속성 컨텍스트에 의해 관리되는 Entity들의 변경사항이 1차 캐시에 저장되고, 이를 flush() 하게되면 데이터베이스에 반영되는 것으로 알고있습니다.

이 때(데이터베이스에 반영될 때), 쓰기 지연 SQL 저장소에 저장되어있던 SQL 쿼리들도 같이 실행되어 데이터베이스에 반영되는 것인가요 ??

제가 이해한 것 :
> flush() -> 
1차 캐시에 있던 엔티티 정보들과 쓰기 지연 SQL 저장소에 저장된 쿼리들로 데이터베이스에 반영
